src/Magnum/ImGuiIntegration/Test/WidgetsGLTest.cpp

```diff
@@ -107,7 +107,13 @@ void WidgetsGLTest::imageButton() {
 
     c.newFrame();
 
-    ImGuiIntegration::imageButton(texture, {100, 100});
+    ImGuiIntegration::imageButton("button", texture, {100, 100},
+        {{}, Vector2{1.0f}}, Color4::yellow(), Color4::blue());
+
+    #ifdef MAGNUM_BUILD_DEPRECATED
+    ImGuiIntegration::imageButton(texture, {100, 100},
```

Comment:
... and then you need to wrap this with `CORRADE_IGNORE_DEPRECATED_PUSH` and `_POP` to not warn when being compiled.